.circleci/config.yml

```diff
@@ -51,5 +51,5 @@ workflows:
       - run-tests:
           matrix:
             parameters:
-              go-version: ["1.14.2"]
+              go-version: ["1.15.15"]
```

Comment:
Yeah... Apparently the url#Redacted() method was introduced in go 1.15; so it makes this a bigger change than I'd anticipated.

I guess I can just copy go's implementation aside if bumping the go version is unacceptable (understandably, of course)

Reply:
Reverted it and just copied the implementation, no need to potentially break existing users